src/Sql/dbo/Stored Procedures/Organization_UpdateStorage.sql

```diff
@@ -35,16 +36,31 @@ BEGIN
             #OrgStorageUpdateTemp
     )
     SELECT
-        @Storage = SUM([Size])
+        @AttachmentStorage = SUM([Size])
     FROM
         [CTE]
 
     DROP TABLE #OrgStorageUpdateTemp
 
+    ;WITH [CTE] AS (
+        SELECT
+            [Id],
+            CAST(JSON_VALUE([Data],'$.Size') AS BIGINT) [Size]
+        FROM
+            [Send]
+        WHERE
+            [UserId] IS NULL
+            AND [OrganizationId] = @Id
+    )
+    SELECT
+        @SendStorage = SUM([CTE].[Size])
+    FROM
+        [CTE]
+
     UPDATE
         [dbo].[Organization]
     SET
-        [Storage] = @Storage,
+        [Storage] = (@AttachmentStorage + @SendStorage),
```

Comment:
Either of these variables could be `null` if no records exists; I would imagine causing issues when performing this addition as the result would be `null` even if one of the two measures had a valid value.

![image](https://user-images.githubusercontent.com/3904944/97905813-a6cbfa00-1d10-11eb-812c-866d7a04da74.png)

Reply:
Good call, with add `ISNULL` checks